deeppavlov/download.py

```diff
@@ -37,10 +39,15 @@
                          " available on disk.")
 
 
-def get_config_downloads(config_path):
+def get_config_downloads(config: Union[str, dict]):
     dp_root_back = get_deeppavlov_root()
-    config = read_json(config_path)
-    set_deeppavlov_root(config)
+    if isinstance(config, (str, Path)):
+        config = read_json(config)
+        set_deeppavlov_root(config)
+    else:
+        deeppavlov_root = Path('download/')
+        deeppavlov_root.mkdir(exist_ok=True)
+        paths.deeppavlov_root = deeppavlov_root
```

Comment:
This is very questionable. The config could have the `deeppavlov_root` property set but it is ignored here.